CHANGELOG.rst

```diff
@@ -2,11 +2,8 @@
 
 (Organized release by release)
 
-Changes since version May 7th, 2021
------------------------------------
-
-Toolbox
-~~~~~~~
+Changes in version 1.0
```

Comment:
there are no "changes" since this is the first version

Reply:
yes, I wasn't sure how to phrase it, it's the first "named release", but the changelog is not empty...?

Reply:
given that this is release 1, why not going for `features` or `functionality` that way, change logs make more sense moving forward, bc we will be `changing` the initial functionality/feature set